src/PublisherMethods.js

```diff
@@ -61,7 +61,9 @@ module.exports = {
 
         promise.then(function(response) {
             return me.completed(response);
-        }).catch(function(error) {
+        });
+        // IE compatibility - catch is a reserved word - without bracket notation source compilation will fail under IE
+        promise["catch"](function(error) {
```

Comment:
While this is correct, the bracket notation instead of a regular method call looks weird to me.
Maybe we could just pass the error handler as the second argument of `then` and avoid the `catch` shortcut altogther